### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by martinobersteiner**

### CODE REVIEW

**Comment by martinobersteiner** [comment]
picture of landing page:
image
(the `FDT` tab in the preview-panel is the collapsed debug-toolbar and only appears in development)

File: invenio_records_lom/marshmallow/__init__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 """Schemas for marshmallow."""
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import, annotations, print_function
```
Comment (utnapischtim) [inline invenio_records_lom/marshmallow/__init__.py:11]: for curiosity: why is this necessary here?

Comment (martinobersteiner) [reply invenio_records_lom/marshmallow/__init__.py:11]: it allows type-hints to reference not yet defined names (e.g. a class's `.create` method returning an instance of the class itself), which is convenient.
see: https://www.python.org/dev/peps/pep-0563/

plus it's part of the core python language from version 3.11:
https://docs.python.org/3/library/__future__.html

now that I look at that last table..., are `absoulte_import` and `print_function` only necessary for compatibilty with `python2.*`?
we don't support that, do we?

File: invenio_records_lom/fixtures/demo.py [hunk 2]
```diff
@@ -382,26 +380,107 @@ def create_fake_record(fake: Faker):
...
+        "resource_type": resource_type,
     }
 
-    service = current_records_lom.records_service
-    draft = service.create(data=data, identity=fake_identity)
-    service.publish(id_=draft.id, identity=fake_identity)
-
 
-def create_fake_records(number: int, seed: int = 42) -> list:
+def publish_fake_record(fake: Faker):
```
Comment (utnapischtim) [inline invenio_records_lom/fixtures/demo.py:403]: please don't use nested functions in such a context. The outer function publish_fake_record gets to long and therefore difficult to understand.

File: invenio_records_lom/marshmallow/__init__.py (hunk 1 above)
Comment (utnapischtim) [reply invenio_records_lom/marshmallow/__init__.py:11]: yes, we don't support python2.*. those imports can be removed.

yeah but why is the annotations import necessary in __init__.py where no annotations are used?

Comment (martinobersteiner) [reply invenio_records_lom/marshmallow/__init__.py:11]: I wrongly thought that `from __future__ import <thing>` would unlock `<thing>` for all files in the sub-file-tree.
It's for this file only though, so I'll remove it.

File: invenio_records_lom/records/systemfields/results.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        relations = self.record.get("metadata", {}).get("relation", [])
+        queue = list(relations)
+        for relation in queue:
+            kind = relation.get("kind", {})
+            if kind.get("source") == self.source and kind.get("value") == self.value:
+                for identifier in relation.get("resource", {}).get("identifier", []):
+                    if identifier.get("catalog") == self._catalog:
+                        data = func(identifier, attrs) or {}
+                        queue.extend(data.get("metadata", {}).get("relation", []))
+
```
Comment (utnapischtim) [inline invenio_records_lom/records/systemfields/results.py:45]: Suggestion:

```python
for relation in queue:
    kind = relation.get("kind", {})
    if kind.get("source") != self.source or kind.get("value") != self.value:
        continue

    for identifier in relation.get("resource", {}).get("identifier", []):
        if identifier.get("catalog") != self._catalog:
            continue
        data = func(identifier, attrs) or {}
        queue.extend(data.get("metadata", {}).get("relation", []))
```

File: invenio_records_lom/resources/config.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+class LOMRecordResourceConfig(RecordResourceConfig):
+    """LOM Record Resource configuration."""
+
+    blueprint_name = "lom_records"
+    url_prefix = "/lom"
+
+    routes = {
+        "item": "/<pid_value>",
+    }
+    # TODO: fill in the rest of config
```
[the remark this answers is not in the document]
Comment (martinobersteiner) [reply invenio_records_lom/resources/config.py:22]: Yep, `Resource`s create the REST-API, so this will be needed in the immediate future.

[the remark this answers is not in the document]
Comment (utnapischtim) [reply invenio_records_lom/resources/config.py:22]: ok

**Review by utnapischtim — APPROVED** [review]
